### ISSUE TITLE
Digit grouping not working on 2.83

### ISSUE THREAD

**Issue by felipetorrents** [leomoon-studios/leomoon-textcounter#7]
If I enable digit grouping, it wont show any of the separators and it gets the animation stuck at the frame it was enabled. Using windows 10, Blender 2.81, 2.82 and 2.83, text counter 1.3.1 and 1.3.4.

![image](https://user-images.githubusercontent.com/48898378/84810694-b228be00-afd9-11ea-9fb0-c7dfb66986b5.png)

**Comment by leomoon** [comment]
Hello,

I just tested on 2.83 LTS and it's working.
[Sample File](https://filedn.com/lySwaAnkmuoJtXuM6Dq7AmF/dls/text-counter_digit-grouping.blend)

Could you please share your blend file?

**Comment by felipetorrents** [comment]
That's weird, your file works fine. If I factory reset and only enable the addon, it'd still won't work.

[Sample File](https://www.dropbox.com/s/6i430icq3g0x5qw/text%20counter%20digit%20grouping.blend?dl=0)

**Comment by leomoon** [comment]
I was able to reproduce this. The grouping and decimal are evaluated in one line and if decimal is not checked, the grouping will have a None value. We'll fix this soon.

I found a quick fix until this is resolved. With Digit Grouping enabled, you can enable Decimal with decimal amount of 0 and that should make the grouping work for now.

Steps to reproduce:
- Create a new scene
- Add a text object
- Enter number between 4 to 7 digits in the counter (more than 3 digits)
- Try enabling Digit Grouping which will give the error below:
```
Traceback (most recent call last):
  File "C:\Users\LeoMoon\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\leomoon-textcounter.py", line 91, in val_up
    textcounter_update_val(context.object, context.scene)
  File "C:\Users\LeoMoon\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\leomoon-textcounter.py", line 337, in textcounter_update_val
    out = formatPaddingCommas(line)
  File "C:\Users\LeoMoon\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\leomoon-textcounter.py", line 272, in formatPaddingCommas
    comma=',' if props.useDigitGrouping else '').replace('.', '@').replace(',', props.digitSeparator).replace('@', props.decimalSeparator)
ValueError: Invalid format specifier
File "C:\Users\LeoMoon\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\leomoon-textcounter.py", line 90, in val_up
```

**Comment by felipetorrents** [comment]
Thank you! the workaround works well. I'm glad this report helped.

**Comment by leomoon** [comment]
Fixed in version 1.3.5
